src/common/datasource/src/file_format/parquet.rs

```diff
@@ -216,6 +220,21 @@ pub async fn stream_to_parquet(
     Ok(rows_written)
 }
 
+/// Customizes per-column properties.
+fn column_wise_config(
+    mut props: WriterPropertiesBuilder,
+    schema: SchemaRef,
+) -> WriterPropertiesBuilder {
+    // Disable dictionary for timestamp column.
```

Comment:
Could you add some comments to describe why to disable dictionary for time index column?